AccessibilityInsightsForAndroidService/app/src/main/java/com/microsoft/accessibilityinsightsforandroidservice/FocusVisualizer.java

```diff
@@ -24,27 +24,11 @@ public FocusVisualizer(
     this.focusVisualizationCanvas = focusVisualizationCanvas;
   }
 
-  public void HandleAccessibilityRedrawEvent(AccessibilityEvent event) {
-    if (event.getEventType() == AccessibilityEvent.TYPE_WINDOW_CONTENT_CHANGED) {
-      return;
-    }
-
-    if (event.getEventType() == AccessibilityEvent.TYPE_WINDOW_STATE_CHANGED) {
-      this.resetVisualizations();
-      return;
-    }
-
-    if (event.getEventType() == AccessibilityEvent.TYPE_VIEW_SCROLLED) {
-      this.updateDrawingsWithNewCoordinates();
-      return;
-    }
-    if (event.getEventType() == AccessibilityEvent.TYPE_WINDOWS_CHANGED) {
-      this.resetVisualizations();
-      return;
-    }
+  public void refreshHighlights() {
```

Comment:
You'll notice that this is lacking tests in this PR. This is because it is lacking tests in main as well. I figured I'd rather get this PR out than to wait on adding tests for something that is already in main.